src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --colorForLinks: #00acdc;
+}
+
+.menu {
+  padding: 0 50px;
+}
+
+.menu__content {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__box {
+  margin-right: 20px;
+}
+
+.nav__box:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: var(--colorForLinks);
+}
+
+.nav__link.is-active {
```

you can combine the styles
```suggestion
.nav__link:hover,
.nav__link.is-active {
```